react/src/hooks/user/profile/useProfile.js

```
@@ -0,0 +1,140 @@
+            data.append('institution', formData.institution || '');
+            data.append('dept', formData.dept || '');
+            data.append('address', formData.address || '');
+            
+            // Append profile picture if exists
+            if (formData.profile_picture) {
+                data.append('profile_picture', formData.profile_picture);
+            }
+
+            // Log the actual FormData contents
+            for (let pair of data.entries()) {
+                console.log(pair[0] + ': ' + pair[1]);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)